3D Chess/Assets/Scripts/Vector3Entity.cs

```diff
@@ -1,16 +1,14 @@
 ﻿using Realms;
 using UnityEngine;
 
+// Every instance of a `Vector3Entity` represents exactly one `PieceEntity`.
```

Comment:
Rather than "represents", perhaps it's more accurate to say "is owned by".

Reply:
Great, that's the wording I was looking for. 🙃